go.mod

```diff
@@ -7,59 +7,51 @@ require (
 	cloud.google.com/go/storage v1.9.0 // indirect
 	github.com/Azure/azure-pipeline-go v0.2.2 // indirect
 	github.com/Azure/azure-storage-blob-go v0.8.0 // indirect
-	github.com/aws/aws-sdk-go v1.31.13
+	github.com/aws/aws-sdk-go v1.35.24
 	github.com/ghodss/yaml v1.0.0
 	github.com/gofrs/uuid v3.2.0+incompatible
 	github.com/google/wire v0.4.0 // indirect
 	github.com/hashicorp/go-plugin v1.0.1-0.20190610192547-a1bc61569a26 // indirect
 	github.com/mattn/go-ieproxy v0.0.1 // indirect
-	github.com/onsi/ginkgo v1.10.3
-	github.com/onsi/gomega v1.7.1
-	github.com/openebs/api v1.11.1-0.20200629052954-e52e2bcd8339
-	github.com/openebs/maya v0.0.0-20200411140727-1c81f9e017b0
-	github.com/openebs/zfs-localpv v1.4.1-0.20210301182642-6ec49df225fa
+	github.com/onsi/ginkgo v1.15.2
+	github.com/onsi/gomega v1.10.2
+	github.com/openebs/api/v2 v2.3.0
+	github.com/openebs/maya v1.12.1-0.20210416090832-ad9c32f086d5
+	github.com/openebs/zfs-localpv v1.6.1-0.20210504173514-62b3a0b7fe5d
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.5.1 // indirect
-	github.com/sirupsen/logrus v1.5.0
-	github.com/spf13/cobra v1.0.0 // indirect
+	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/objx v0.2.0 // indirect
-	github.com/vmware-tanzu/velero v1.3.2
+	github.com/vmware-tanzu/velero v1.5.0
```

Comment:
can we upgrade it to 1.6.0?

Reply:
1.6.0 has some changes the way client has been used, they are using the controller-runtime client instead of velero-client in some places, so i thought making those many changes at the last minute will not be good idea.